src/fmu/ensemble/virtualrealization.py

```diff
@@ -595,3 +559,40 @@ def parameters(self):
     def name(self):
         """Return name of ensemble"""
         return self._description
+
+
+def smry_cumulative(column_keys):
+    """Determine whether smry vectors are cumulative
+
+    Returns list of booleans, indicating whether a certain
+    column_key in summary dataframes corresponds to a cumulative
+    column.
+
+    The current implementation checks for the letter 'T' in the
+    column key, but this behaviour is not guaranteed in the
+    future, in case the cumulative information gets internalized
+
+    Since the current implementation might not be reliable (but
+    sufficient for use in the interpolation code), it is not
+    exposed as a public API.
```

Comment:
Isn't this comment less valid now after refactoring from _smry_cumulative to smry_cumulative? Not more reliable, but more exposed?

Reply:
Thanks, corrected. It can be considered to change this after merging of #89